flink-connectors/flink-connector-kafka-0.8/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer08.java

```diff
@@ -187,31 +191,65 @@ public FlinkKafkaConsumer08(List<String> topics, KeyedDeserializationSchema<T> d
 		validateAutoOffsetResetValue(props);
 	}
 
+	/**
+	 * Search offset from timestamp for each topic in kafka. If no offset exist, use the latest offset.
+	 *
+	 * @param partitionTimesMap Kafka topic partition and timestamp
+	 * @return Kafka topic partition and the earliest offset after the timestamp. If no offset exist, use the latest offset in kafka
+	 */
+	private Map<KafkaTopicPartition, Long> convertTimestampToOffset(Map<KafkaTopicPartition, Long> partitionTimesMap) {
```

Comment:
Actually I think lets just disable the timestamp option for 0.8.

I just think its a bit strange that the functionality is there for 0.8 and 0.10, but skipped for 0.10.

Sorry for jumping back and forth here, trying to figure out what would be most natural.